set.mm

```diff
@@ -476582,24 +476577,24 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
        Arnoux, 29-Dec-2019.) $)
     txomap $p |- ( ph -> ( H " A ) e. ( L tX M ) ) $=
       ( vc va vb vz cima ctx co wcel cxp wss wrex wral cfv wceq simp-6l simpllr
-      cv wa syl2anc simplr wfn cop opex fnmpoi a1i ctopon toponss xpss12 simprl
-      syl fnfvima syl3anc simp-4r wf ffn 3syl fimaproj imass2 ad2antll eqsstrrd
+      cv wa syl2anc simplr wfn opex fnmpoi ctopon ad6antr toponss xpss12 simprl
+      cop fnfvima mp3an2i simp-4r wf ffn 3syl fimaproj imass2 ad2antll eqsstrrd
       3eltr3d xpeq1 eleq2d sseq1d anbi12d xpeq2 rspc2ev syl112anc wb eltx mpbid
-      r19.21bi adantlr adantr r19.29vva wfun mpofun fvelima mpan adantl r19.29a
-      ralrimiva mpbird ) AHDUJZKLUKULUMZUFVBZUGVBZUHVBZUNZUMZXNXIUOZVCZUHLUPUGK
-      UPZUFXIUQZAXRUFXIAXKXIUMZVCZUIVBZHURZXKUSZXRUIDYAYBDUMZVCZYDVCZYBBVBZCVBZ
-      UNZUMZYJDUOZVCZXRBCIJYGYHIUMZVCZYIJUMZVCZYMVCZFYHUJZKUMZGYIUJZLUMZXKYSUUA
-      UNZUMZUUCXIUOZXRYRAYNYTAXTYEYDYNYPYMUTZYGYNYPYMVAZUBVDYRAYPUUBUUFYOYPYMVE
-      ZUCVDYRYCHYJUJZXKUUCYRHMNUNZVFZYJUUJUOZYKYCUUIUMUUKYRBCMNYHFURZYIGURZVGZH
-      UEUUMUUNVHVIVJYRYHMUOZYINUOZUULYRIMVKURZUMZYNUUPYRAUUSUUFRVOUUGYHIMVLVDZY
-      RJNVKURZUMZYPUUQYRAUVBUUFSVOUUHYIJNVLVDZYHMYINVMVDYQYKYLVNUUJYJHYBVPVQYFY
-      DYNYPYMVRYRBCMNFGHYHYIUEYRAMOFVSFMVFUUFPMOFVTWAYRANEGVSGNVFUUFQNEGVTWAUUT
-      UVCWBZWFYRUUCUUIXIUVDYLUUIXIUOYQYKYJDHWCWDWEXQUUDUUEVCXKYSXMUNZUMZUVEXIUO
-      ZVCUGUHYSUUAKLXLYSUSZXOUVFXPUVGUVHXNUVEXKXLYSXMWGZWHUVHXNUVEXIUVIWIWJXMUU
-      AUSZUVFUUDUVGUUEUVJUVEUUCXKXMUUAYSWKZWHUVJUVEUUCXIUVKWIWJWLWMYFYMCJUPBIUP
-      ZYDAYEUVLXTAUVLUIDADIJUKULUMZUVLUIDUQZUDAUUSUVBUVMUVNWNRSBCDIJUURUVAUIWOV
-      DWPWQWRWSWTXTYDUIDUPZAHXAXTUVOBCMNUUOHUEXBUIXKDHXCXDXEXFXGAKOVKURZUMLEVKU
-      RZUMXJXSWNTUAUGUHXIKLUVPUVQUFWOVDXH $.
+      r19.21bi ad4ant13 r19.29vva wfun mpofun fvelima adantl r19.29a ralrimiva
+      mpan mpbird ) AHDUJZKLUKULUMZUFVBZUGVBZUHVBZUNZUMZXLXGUOZVCZUHLUPUGKUPZUF
+      XGUQZAXPUFXGAXIXGUMZVCZUIVBZHURZXIUSZXPUIDXSXTDUMZVCZYBVCZXTBVBZCVBZUNZUM
+      ZYHDUOZVCZXPBCIJYEYFIUMZVCZYGJUMZVCZYKVCZFYFUJZKUMZGYGUJZLUMZXIYQYSUNZUMZ
+      UUAXGUOZXPYPAYLYRAXRYCYBYLYNYKUTZYEYLYNYKVAZUBVDYPAYNYTUUDYMYNYKVEZUCVDYP
+      YAHYHUJZXIUUAHMNUNZVFYPYHUUHUOZYIYAUUGUMBCMNYFFURZYGGURZVNZHUEUUJUUKVGVHY
+      PYFMUOZYGNUOZUUIYPIMVIURZUMZYLUUMAUUPXRYCYBYLYNYKRVJUUEYFIMVKVDZYPJNVIURZ
+      UMZYNUUNAUUSXRYCYBYLYNYKSVJUUFYGJNVKVDZYFMYGNVLVDYOYIYJVMUUHYHHXTVOVPYDYB
+      YLYNYKVQYPBCMNFGHYFYGUEYPAMOFVRFMVFUUDPMOFVSVTYPANEGVRGNVFUUDQNEGVSVTUUQU
+      UTWAZWEYPUUAUUGXGUVAYJUUGXGUOYOYIYHDHWBWCWDXOUUBUUCVCXIYQXKUNZUMZUVBXGUOZ
+      VCUGUHYQYSKLXJYQUSZXMUVCXNUVDUVEXLUVBXIXJYQXKWFZWGUVEXLUVBXGUVFWHWIXKYSUS
+      ZUVCUUBUVDUUCUVGUVBUUAXIXKYSYQWJZWGUVGUVBUUAXGUVHWHWIWKWLAYCYKCJUPBIUPZXR
+      YBAUVIUIDADIJUKULUMZUVIUIDUQZUDAUUPUUSUVJUVKWMRSBCDIJUUOUURUIWNVDWOWPWQWR
+      XRYBUIDUPZAHWSXRUVLBCMNUULHUEWTUIXIDHXAXEXBXCXDAKOVIURZUMLEVIURZUMXHXQWMT
+      UAUGUHXGKLUVMUVNUFWNVDXF $.
```

Comment:
This is in TA's mathbox.  The proof shrinks from 65 down to 6 lines.  No change in Syntax, Axioms and Definitions usage.